packages/web/src/components/modal-radio/ModalRadioItem.module.css

```diff
@@ -27,8 +20,7 @@
 }
 
 .input {
-  height: 24px;
-  width: 24px;
+  /* placeholder */
```

Comment:
delete?

Reply:
so that weirdly needs to be their to have styling work, but when we migrate to radio + icon + label in harmony this can go away

Reply:
:o that's wildin

Reply:
maybe add a comment so other people don't delete lol

Reply:
it's because of the selectors in the other styles below like `.root:has(.input:checked)`

~~These are all broken on Firefox and shouldn't be used~~ whoa wait Firefox supports `:has()` now 🥳 https://www.mozilla.org/en-US/firefox/121.0/releasenotes/

YESS we can finally get rid of all the focus state tracking etc!